coralproject/talk-fe
# Adding sample article

## What does this PR do?

Adds a sample article at ./samplearticle.html to view a comment stream in context.

## How do I test this PR?

1. Navigate to `talk-fe/packages/coral-embed-stream`
2. `npm run build`
3. Navigate to `dist` directory and start a server (you can do this with `python -m SimpleHTTPServer 61685`)
4. Check a web browser (The url will be `http://localhost:61685/samplearticle.html` if you used the above command.) You should see a comment stream and be able to interact with it.

## Note

For some reason `pym.js`, which we’re using to manage iframe height, stopped working as advertised. A patch has been added to set iframe height to match that of the comment stream.
